### ISSUE TITLE
Argument is deprecated

### ISSUE THREAD

**Issue by manfred0191** [Azure/terraform-azurerm-caf-enterprise-scale#308]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Versions

**terraform**:

**azure provider**:

**module**: 1.1.3

### Description

**Describe the bug**

**Steps to Reproduce**

1. step 1
2. step 2
3. you get it...

**Screenshots**

**Additional context**

Hello,

everytime I deploy things with the model I get the error message- in every deployment:

```
│ Warning: Argument is deprecated
│ 
│   with module.enterprise_scale.azurerm_policy_definition.enterprise_scale,
│   on .terraform/modules/enterprise_scale/resources.policy_definitions.tf line 12, in resource "azurerm_policy_definition" "enterprise_scale":
│   12:   management_group_name = try(basename(each.value.scope_id), null)
│ 
│ Deprecated in favour of `management_group_id`
│ 
│ (and 111 more similar warnings elsewhere)
```

Basically I thought this behavior would be fixed with the new version as I was running module 1.0.0 before, but seemingly this has not yet been done. Therefore I open up this issue to remind you that there is a change to be done.

Thx
Manfred

**Comment by jtracey93** [comment]
Hey @manfred0191,

Thanks for raising this we are fixing this in the upcoming release of v1.2.0 👍

This has been tracked in issue #272 👍

Thanks

Jack
